python/tvm/relay/op/strategy/arm_cpu.py

```diff
@@ -198,7 +231,9 @@ def conv2d_strategy_arm_cpu(attrs, inputs, out_type, target):
         elif layout == "NHWC":
             assert kernel_layout == "HWOI"
             strategy.add_implementation(
-                wrap_compute_conv2d(topi.arm_cpu.compute_depthwise_conv2d_nhwc),
+                wrap_compute_conv2d(
+                    topi.arm_cpu.compute_depthwise_conv2d_nhwc, need_auto_scheduler_layout=True
```

Comment:
1. I think we don't have official support for depthwise convolution for layout rewrite. 
2. I prefer using `auto_scheduler_enabled` to distinguish and just call `topi.nn.depthwise`:
```python
        elif layout == "NHWC":
            assert kernel_layout == "HWOI"
            from tvm.auto_scheduler import is_auto_scheduler_enabled
            if is_auto_scheduler_enabled():
                strategy.add_implementation(
                    wrap_compute_conv2d(topi.nn.depthwise_conv2d_nhwc),
                    naive_schedule,
                    name="depthwise_conv2d_nhwc.arm_cpu",
                )
            else:
                strategy.add_implementation(
                    wrap_compute_conv2d(topi.arm_cpu.compute_depthwise_conv2d_nhwc),
                    wrap_topi_schedule(topi.arm_cpu.schedule_depthwise_conv2d_nhwc),
                    name="depthwise_conv2d_nhwc.arm_cpu",
                )
```

Reply:
If AutoSchedule is enabled, use `wrap_topi_schedule(topi.arm_cpu.schedule_depthwise_conv2d_nhwc)` is the same as `naive_schedule`. So the then path and the else path are the same.

This will only take effect if auto_schedule is not enabled.

Reply:
> If AutoSchedule is enabled, use `wrap_topi_schedule(topi.arm_cpu.schedule_depthwise_conv2d_nhwc)` is the same as `naive_schedule`. So the then path and the else path are the same.
> 
> This will only take effect if auto_schedule is not enabled.

My concern is `topi.arm_cpu.compute_depthwise_conv2d_nhwc` maybe changed for auto tvm template (like spatial pack). However, auto scheduler's compute template should be generic computation. The way I mentioned could guard it.